src/node.cc

```diff
@@ -4556,7 +4557,9 @@ inline int Start(Isolate* isolate, IsolateData* isolate_data,
   {
     SealHandleScope seal(isolate);
     bool more;
+    PERFORMANCE_MARK(&env, LOOP_START);
     do {
+      PERFORMANCE_MARK(&env, LOOP_FRAME);
```

Comment:
loop frame is expected to measure the start of the current loop iteration. I noticed the changes there so I needed to go back and check if this needed to be redone.